packages/@storefront/flux-capacitor/src/core/reducers/data/past-purchases.ts

```diff
@@ -27,8 +27,15 @@ export type Action = Actions.ReceivePastPurchaseSkus
   | Actions.ReceivePastPurchaseTemplate;
 export type State = Store.PastPurchase;
 
-export enum SORT_ENUMS {
-  DEFAULT, MOST_PURCHASED, MOST_RECENT
+export namespace PAST_PURCHASE_SORTS {
+  export const DEFAULT = 'Default';
+  export const MOST_PURCHASED = 'Most Purchased';
+  export const MOST_RECENT = 'Most Recent';
+  export const ALL = [
```

Comment:
Might want to potentially add some sort of leading identifier so people know that this is a collection of the other properties, but I am fine if it says as is. Thoughts @Vic-Dev @jmbtutor?

Reply:
Dropped the `ALL` key entirely. Resolving.